hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AbstractFSWAL.java

```diff
@@ -686,12 +686,6 @@ public void abortCacheFlush(byte[] encodedRegionName) {
     this.sequenceIdAccounting.abortCacheFlush(encodedRegionName);
   }
 
-  @Override
-  public long getEarliestMemStoreSeqNum(byte[] encodedRegionName) {
-    // Used by tests. Deprecated as too subtle for general usage.
-    return this.sequenceIdAccounting.getLowestSequenceId(encodedRegionName);
```

Comment:
Does this method in SequenceIdAccounting still useful?

Reply:
This method is only being called by UTS, and I will attempt to remove those calls.

Reply:
That's why I say we need to introduce a equivalent version of this method in test code.